core/templates/dev/head/components/summary_tile/collection_summary_tile_directive.html

```diff
@@ -33,37 +30,18 @@ <h2 class="activity-title"><[getCollectionTitle()]></h2>
               </li>
             </ul>
           </div>
-          <div class="title-section-mask top-card-mask"></div>
+          <div class="title-section-mask"></div>
         </a>
       </md-card>
-      <div class="title-section-mask"></div>
-    </md-card>
-    <div class="title-section-mask"></div>
-  </md-card>
 </script>
 
 <style>
-  .bottom-card {
-    box-shadow: 8px 8px 2px 0 rgba(0, 0, 0, 0.26);
-    height: 250px;
-    margin: 12px 10px 12px 4px;
-  }
-  .bottom-card .title-section {
-    height: 154px;
-  }
-
-  .middle-card, .top-card {
-    box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26);
-    height: 250px;
-    margin: -150px 3px;
-    z-index: 11;
-  }
-
-  .top-card .thumbnail-image {
-    top: 11.5%;
+  .thumbnail-image {
```

Comment:
Could you perhaps prefix this as follows:

    .oppia-activity-summary-tile .thumbnail-image

so that it doesn't affect other elements in the app?